src/bank.rs

```diff
@@ -187,6 +188,22 @@ impl Default for Bank {
 }
 
 impl Bank {
+    /// Create an Bank with built-in programs.
+    pub fn new_with_builtin_programs() -> Self {
+        let bank = Self::default();
+
+        {
+            // Preload Bpf Loader account
+            let mut accounts = bank.accounts.write().unwrap();
+            let mut account = accounts
+                .entry(bpf_loader::id())
+                .or_insert_with(Account::default);
+            bpf_loader::populate_account(&mut account);
+        }
+
+        bank
+    }
+
     /// Create an Bank using a deposit.
     pub fn new_from_deposit(deposit: &Payment) -> Self {
         let bank = Self::default();
```

Comment:
oh can we use `Self::new_with_builtin_programs()` here now, to de-dup the "Preload Bpf Loader account" block?